services/api.js

```diff
@@ -128,11 +128,13 @@ export default class Api {
       this.addParam(url, this.shopApi?.cost?.totalShippingAmount?.current?.amount, "shipping_price");
 
       // Address information (may fail if shop doesn't have access)
-      this.addParam(url, this.shopApi?.shippingAddress?.current?.firstName, "first_name");
-      this.addParam(url, this.shopApi?.shippingAddress?.current?.countryCode, "country_code");
-      this.addParam(url, this.shopApi?.shippingAddress?.current?.zip, "zip");
+      this.addParam(url, this.shopApi?.shippingAddress?.current?.firstName || this.shopApi?.billingAddress?.current?.firstName, "first_name");
+      this.addParam(url, this.shopApi?.shippingAddress?.current?.countryCode || this.shopApi?.billingAddress?.current?.countryCode, "country_code");
+      this.addParam(url, this.shopApi?.shippingAddress?.current?.zip || this.shopApi?.billingAddress?.current?.zip, "zip");
 
       this.addParam(url, this.shopApi?.extension?.target, "target");
+      this.addParam(url, this.shopApi?.extension?.version, "script_version");
+      this.addParam(url, this.shopApi?.extension?.apiVersion, "api_version");
```

Comment:
We may want to start giving these shopify specific names.. script_version and api_version have different meanings to our platform. Target is also pretty generic and I was renaming that to shopify_app_target for reporting.